query/lambda/lambda_function_use_latest_runtime.sql

```diff
@@ -11,6 +11,8 @@ select
   end as reason,
   -- Additional Dimensions
   region,
-  account_id
+  account_id,
+  packagetype
 from
-  aws_lambda_function;
+  "aws_lambda_function"
+where packagetype = 'Zip';
```

Comment:
```suggestion
```

In general, we like to show all resources when writing controls so users know that we are evaluating all resources (even if we use the `skip` state). For this control, I think it would actually be better if we handle this case by checking if `runtime` is `null`, and if so, set the state to `skip` and `reason` to `title || ' uses custom runtime.'`

This lets the user know we're:
1. Checking every Lambda function
2. Verified that for Lambda functions of type `Image` there is no runtime defined, so we're skipping the runtime check for them

This aligns with the [AWS Foundational Security Best Practices control](https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-standards-fsbp-controls.html#fsbp-lambda-1), which states: `The AWS Config rule ignores functions that have a package type of Image.`. For a Steampipe control, this means we'd use the `skip` state.

For the query structure, I recommend looking at https://github.com/turbot/steampipe-mod-aws-compliance/blob/main/query/ebs/ebs_attached_volume_delete_on_termination_enabled.sql as an example, as this handles a skip/ok/alarm scenario similar to this one.